### ISSUE TITLE
[FEA] Series with MultiIndex doesn't support tuple indexing

### ISSUE THREAD

**Issue by asmeurer** [rapidsai/cudf#2189]
**Is your feature request related to a problem? Please describe.**
```py
>>> df = pandas.DataFrame({'a': [0, 0, 1], 'b': [0, 1, 0], 'c': [0, 1, 2]})
>>> df.set_index(['a', 'b'], inplace=True)
>>> cdf = cudf.from_pandas(df)
>>> df['c'][(0, 0)]
0
>>> df['c'].loc[(0, 0)]
0
>>> cdf['c'][(0, 0)]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aaronmeurer/anaconda3/lib/python3.7/site-packages/cudf/dataframe/series.py", line 304, in __getitem__
    raise NotImplementedError(type(arg))
NotImplementedError: <class 'tuple'>
>>> cdf['c'].loc[(0, 0)]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aaronmeurer/anaconda3/lib/python3.7/site-packages/cudf/indexing.py", line 48, in __getitem__
    type(arg).__name__
NotImplementedError: .loc not implemented for label type tuple
>>> cudf.__version__
'0.8.0+0.g8fa7bd3.dirty'
```

**Describe the solution you'd like**
It should work the same as pandas.

**Describe alternatives you've considered**
I haven't been able to find any workarounds, aside from converting the dataframe to pandas.

**Comment by thomcom** [comment]
Hi! You should be able to index with the tuples first into the Dataframe, then into the Series:

`cdf.loc[(0, 0)]['c']`

**Comment by thomcom** [comment]
In fact for your particular example just `cdf.loc[(0,0)]`. MI indexing into Series is an upcoming feature, but it already works on dataframes!

**Comment by kkraus14** [comment]
This is resolved as of 0.14.
